perfkitbenchmarker/configs/storage_utility.py

```diff
@@ -0,0 +1,173 @@
+                                     self._cpu_count,
+                                     self._number_disks)
+    print vm_config
+
+  def SetStorageSize(self):
+    """Set Storage size.
+
+    GCP: ratings from https://cloud.google.com/compute/docs/disks/. Storage can
+          go as high as 64TB per disk but IOPS maxes out at 30,000 iops/disk
+          or (1000GB).
+    AWS: ratings from
+          http://docs.aws.amazon.com/AWSEC2/latest/
+          UserGuide/EBSVolumeTypes.html#EBSVolumeTypes_gp2. Storage can go as
+          high as 16TiB per disk but IOPS maxes out at 10000 IOPS/volume size.
+          Reference gives volume size in GiB so converted to GB below.
+    """
+    if self._provider == self.GCP:
+      self._storage_size = min(self._iops / 30, 30000 / 30)
+    elif self._provider == self.AWS:
+      value = self._iops
+      value = numpy.array(value)
+      self._storage_size = int(numpy.piecewise(value,
+                                               [[value <= 100],
+                                                [(value > 100) &
+                                                 (value <= 9999)],
+                                                [value > 9999]],
+                                               [lambda x:
+                                                int(math.ceil(1.07374)),
+                                                lambda x:
+                                                int(math.ceil(3 * value)),
+                                                lambda x:
+                                                int(math.ceil(3579.855))]))
+
+  def GetStorageSize(self):
+    """Return storage size.
+    Returns:
+      _storage_size: Storage size (GB).
+    """
+    return self._storage_size
+
+  def SetCPUCount(self):
+    """Set cpu count.
+
+    GCP: ratings from
+    https://cloud.google.com/compute/docs/disks/performance#ssd-pd-performance
+    AWS: ratings from https://aws.amazon.com/ebs/details/
+    """
+    if self._provider == self.GCP:
+      value = self._iops
+      self._cpu_count = int(numpy.piecewise(value,
+                                            [[value <= 15000],
+                                             [(value > 15000) &
+                                              (value <= 25000)],
+                                             [value > 25000]],
+                                            [lambda x: 1,
+                                             lambda x: 16,
+                                             lambda x: 32]))
+    elif self._provider == self.AWS:
+      self._cpu_count = 1
+
+  def GetCPUCount(self):
+    """Return CPU count.
+
+    Returns:
+      _cpu_count: CPU count.
+    """
+    return self._cpu_count
+
+  def SetNumberDisks(self):
+    """Set number of disks.
+
+    GCP: ratings from https://cloud.google.com/compute/docs/disks/performance
+    AWS: ratings from https://aws.amazon.com/ebs/details/
+    """
+    if self._provider == self.GCP:
```

Comment:
For GCP, there is no need to use multiple disks to achieve higher iops. Adding disks don't help increase IOPS.